data/json/items/melee.json

```diff
@@ -2152,9 +2152,9 @@
     "material": [ "aluminum", "wood" ],
     "techniques": [ "SWEEP" ],
     "volume": 6,
-    "bashing": 2,
+    "bashing": 8,
     "cutting": 1,
-    "flags": [ "SPEAR", "NONCONDUCTIVE" ],
+    "flags": [ "SPEAR", "NONCONDUCTIVE", "FRAGILE_MELEE" ],
```

Comment:
If we have an indication that the head would just be soldered on, fragile is warranted, I dont know if that's typical.